tests/Microsoft.DotNet.Framework.Docker.Tests/ImageTests.cs

```diff
@@ -75,20 +75,30 @@ private static string GetFilterRegexPattern(string filter)
         }
 
         [Theory]
+        [Trait("Category", "Runtime")]
         [MemberData(nameof(GetVerifyImagesData))]
         public void VerifyImagesWithApps(ImageDescriptor imageDescriptor)
         {
-            VerifyImages(imageDescriptor, "dotnetapp", "", true);
+            VerifyFxImages(imageDescriptor, "dotnetapp", "", true);
         }
 
         [Theory]
+        [Trait("Category", "Runtime")]
         [MemberData(nameof(GetVerifyImagesData))]
         public void VerifyImagesWithWebApps(ImageDescriptor imageDescriptor)
         {
-            VerifyImages(imageDescriptor, "webapp", "powershell -command \"dir ./bin/SimpleWebApplication.dll\"", false);
+            VerifyFxImages(imageDescriptor, "webapp", "powershell -command \"dir ./bin/SimpleWebApplication.dll\"", false);
         }
 
-        private void VerifyImages(ImageDescriptor imageDescriptor, string appDescriptor, string runCommand, bool includeRuntime)
+        [Theory]
+        [Trait("Category", "ASPNET")]
+        [MemberData(nameof(GetVerifyImagesData))]
+        public void VerifyAspnetImagesWithApps(ImageDescriptor imageDescriptor)
+        {
+            VerifyAspnetImages(imageDescriptor, "aspnet", "", true);
```

Comment:
The `ImageDescriptor` that is passed in here is essentially a matrix consisting of runtime/SDK/OS values.  The test gets executed for each combination that is defined in the `TestData` field.  The problem with this is that the ASP.NET tests do not even care about the SDK version.  The ASP.NET tests are only using the runtime/OS values.  So the ASP.NET tests end up processing multiple `ImageDescriptors` that consist of the same values for runtime/OS which amounts to running a duplicate test for no reason.

For example, there exists this snippet from the `TestData` field:
```
new ImageDescriptor { RuntimeVersion = "4.7.2", BuildVersion = "4.7.2", OsVariant = WSC_LTSC2019 },
new ImageDescriptor { RuntimeVersion = "4.7.2", BuildVersion = "4.8", OsVariant = WSC_LTSC2019 },
```

The only variation between those is the BuildVersion property.  The ASP.NET tests don't consume this property  This means the ASP.NET tests run a test for `RuntimeVersion = 4.7.2` and `OS = WSC_LTSC2019` twice!

To fix this, I think ASP.NET should have its own set of ImageDescriptors that define the matrix of values it cares about, separate from the existing tests.